test/zdtm/static/cgroup04.checkskip

```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
+set -e
 
-! test -f /sys/fs/cgroup/cgroup.controllers
+test ! -f /sys/fs/cgroup/cgroup.controllers
+
+grep -q '^devices\s' /proc/cgroups
+grep -q '^memory\s' /proc/cgroups
+
+if grep -q ' cgroup rw,.*devices' /proc/mounts; then
```

Comment:
I think it is better to grep /proc/self/cgroup. The test doesn't use existing mounts and we need to check that this controller isn't combined with others.
I think it should be like this:
```
grep '^[0-9]*:devices:' /proc/self/cgroup
```

and pls add a comment that tells what we are checking here.